master/internal/core.go

```diff
@@ -1110,7 +1110,7 @@ func (m *Master) Run(ctx context.Context) error {
 	m.echo.File("/api/v1/api.swagger.json",
 		filepath.Join(m.config.Root, "swagger/determined/api/v1/api.swagger.json"))
 
-	m.echo.GET("/config", api.Route(m.getConfig))
+	// m.echo.GET("/config", api.Route(m.getConfig))
```

Comment:
could this method also get removed?

Reply:
or moved into the api_ layer
m.getConfig

Reply:
Fair point, I had left it in because it looked like it was potentially getting used elsewhere, I'll check and see if we can completely remove it; otherwise, I'll move it to the api layer

Reply:
Got it removed!

Reply:
great! the go compiler should be good at complaining about it if it's used elsewhere what I'd be more worried about is if /config is called somewhere we missed.